doc/api/cli.md

```diff
@@ -1292,6 +1292,15 @@ Specify ways of the inspector web socket url exposure.
 By default inspector websocket url is available in stderr and under `/json/list`
 endpoint on `http://host:port/json/list`.
 
+### `--inspect-wait[=[host:]port]`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Activate inspector on `host:port` and wait for debugger to be attached.
+Default `host:port` is `127.0.0.1:9229`.
```

Comment:
The docs should likely include additional detail explaining the difference between this and `--inspect-brk`. Just looking at this I would have no idea as an unfamiliar user which to choose and why.